xUnit.net v3 In-Process Runner v4.0.2+0c7aa4991f (64-bit .NET 10.0.13)
  Discovering: Pzd.SharedKernel.UnitTests
  Discovered:  Pzd.SharedKernel.UnitTests
  Starting:    Pzd.SharedKernel.UnitTests
    Pzd.SharedKernel.UnitTests.ValueObjectTests.分量相同但类型不同的值对象不相等 [FAIL]
      Assert.False() Failure
      Expected: False
      Actual:   True
      Stack Trace:
        tests\Pzd.SharedKernel.UnitTests\ValueObjectTests.cs(40,0): at Pzd.SharedKernel.UnitTests.ValueObjectTests.分量相同但类型不同的值对象不相等()
           at System.Reflection.MethodBaseInvoker.InterpretedInvoke_Method(Object obj, IntPtr* args)
           at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
  Finished:    Pzd.SharedKernel.UnitTests (ID = '589d5d70a300589d2a41ee2f939cdeb50feade9e9abc553a2272c43f4ef292f9')
=== TEST EXECUTION SUMMARY ===
   Pzd.SharedKernel.UnitTests  Total: 25, Errors: 0, Failed: 1, Skipped: 0, Not Run: 0, Time: 0.119s
